src/services/ui/src/components/Layout/index.tsx

```diff
@@ -9,9 +9,11 @@ import { AwsCognitoOAuthOpts } from "@aws-amplify/auth/lib-esm/types";
 import { Footer } from "../Footer";
 import { UsaBanner } from "../UsaBanner";
 import { FAQ_TARGET } from "@/routes";
+import { useUserContext } from "../Context/userContext";
+import { useOsQuery } from "../Opensearch";
 
-const getLinks = (isAuthenticated: boolean) => {
-  if (isAuthenticated) {
+const getLinks = (isAuthenticated: boolean, role?: string) => {
+  if (isAuthenticated && role !== "onemac-micro-statesubmitter") {
```

Comment:
Are we sure this is right? State Submitters will need Dashboard access as far as I know

Reply:
Thanks Keven, This is absolutely not the role we want, i was using this to test. i have a meet to mike at 10. This was what i needed is thought on regarding users. this is also causing e2e to fail because the test is looking for the dashboard to indicate the user is logged in